packages/protocol/test/bridge/TokenVault.test.ts

```diff
@@ -1,94 +1,135 @@
-import { expect } from "chai";
-import { AddressManager, TokenVault } from "../../typechain";
-import { ethers } from "hardhat";
-import { BigNumber, BigNumberish } from "ethers";
-import { ADDRESS_RESOLVER_DENIED } from "../constants/errors";
+/* eslint-disable camelcase */
+import * as helpers from "@nomicfoundation/hardhat-network-helpers"
+import { expect } from "chai"
+import {
+    AddressManager,
+    AddressManager__factory,
+    BridgedERC20,
+    TestMessageSender__factory,
+    TokenVault,
+    TokenVault__factory,
+} from "../../typechain"
+import hre, { ethers } from "hardhat"
+import { BigNumber, BigNumberish } from "ethers"
+import { getSlot } from "../../tasks/utils"
+import { ADDRESS_RESOLVER_DENIED } from "../constants/errors"
+import { smock } from "@defi-wonderland/smock"
 
 type CanonicalERC20 = {
-    chainId: BigNumberish;
-    addr: string;
-    decimals: BigNumberish;
-    symbol: string;
-    name: string;
-};
+    chainId: BigNumberish
+    addr: string
+    decimals: BigNumberish
+    symbol: string
+    name: string
+}
 
 const weth: CanonicalERC20 = {
     chainId: 5,
     addr: "0xB4FBF271143F4FBf7B91A5ded31805e42b2208d6",
     decimals: 18,
     symbol: "WETH",
     name: "Wrapped Ether",
-};
```

Comment:
not quite sure, but seems using semicolons is recommended? ref: https://github.com/taikoxyz/taiko-mono/pull/480

Reply:
I'll go back and change it!